app/javascript/stylesheets/application.css

```diff
@@ -18,14 +18,23 @@
 
   h3,
   h4 {
-    @apply text-3xl md:text-4xl 2xl:text-5xl font-bold !important;
+    @apply text-3xl md:text-4xl 2xl:text-5xl !important;
   }
 
   h5,
   h6 {
     @apply text-2xl md:text-3xl 2xl:text-4xl !important;
   }
 
+  h3,
```

Comment:
Please, create classes instead of styling HTML elements.

Reply:
Oh ok, but we didn't make this file, it comes from another previously merged branch 😅

Reply:
We'll implement this in another issue